ample/util/printTable.py

```diff
@@ -2,7 +2,8 @@
 
 import locale
 import sys
-locale.setlocale(locale.LC_NUMERIC, "")
+# Below causes issues in some locales and noone knows why it was included so commenting out for now
```

Comment:
I think we should delete this once and for all. Otherwise somebody like @hlasimpk might uncomment it in the future and there we go again ;-)

Reply:
Good point, but it might have been there for a reason and it could help with future debugging so I'm tempted to leave it and just deny @hlasimpk access to the repository. Or from github? ;)

Reply:
I think the internet would be most suitable